config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :create] do
```

Comment:
- [ ] It would be better if we have `GET /posts/new` in our routes so that we can return a new object

Reply:
so basically instead of create do i change it to new :confused:

Reply:
> so basically instead of create do i change it to new confused

@Cluab, that is not what I'm asking you to do. Also the create method uses the POST request and the new method uses the GET request so they are not the same, both have different functions